src/main/java/bio/terra/common/CloudPlatformWrapper.java

```diff
@@ -55,6 +57,10 @@ public boolean isAzure() {
     return false;
   }
 
+  public <T> T choose(Map<CloudPlatform, Supplier<T>> cloudMap) {
+    return cloudMap.get(CloudPlatform.GCP).get();
```

Comment:
Since this is the base class you wouldn't put the GCP version here, it would go in `GcpPlatform` and this would be `abstract`. Another approach is to switch in the base class:
```suggestion
    return cloudMap.get(getCloudPlatform()).get();
```